zebra/if_netlink.c

```diff
@@ -601,11 +604,15 @@ static int netlink_interface(struct sockaddr_nl *snl, struct nlmsghdr *h,
 		desc = (char *)RTA_DATA(tb[IFLA_IFALIAS]);
 
 	if (tb[IFLA_LINKINFO]) {
+		/* if at least among the interfaces, one LINKINFO is present,
+		 * this means VRF lite capable. e.g., kernel does
+		 * support filtering on kind, so userspace needs to handle
+		 */
+		local_linkinfo_presence = 1;
```

Comment:
I don't think this is correct. `IFLA_LINKINFO` has been around for over a decade, its presence certainly doesn't indicate that the kernel supports the VRF device.